src/schemas/graphql-schema.js

```diff
@@ -1,26 +1,33 @@
 'use strict';
 
-const graphql = require('graphql');
+const {
+  GraphQLObjectType,
+  GraphQLSchema,
+  GraphQLID,
+  GraphQLList
+} = require('graphql');
 
+const Product = require('../models/db/product');
```

Comment:
This variable name looks confusing. Change it to something more descriptive like `ProductModel`.
```suggestion
const ProductModel = require('../models/db/product');
```

Reply:
You are right it is quite confusing, but it seems that ommiting the `Model` part it is a common naming convention using mongoose.

For example in the docs (https://mongoosejs.com/docs/guide.html#models), `Blog` and `Animal` are used instead of the more exact `BlogModel` and `AnimalModel`.

Maybe @magnus195 would have some thoughts on this?

Reply:
Right, models don't usually end with `Model`. The reason for it is because of the possibility of making e.g. a Product object instead of a Product model object.